pkg/lib/cockpit-components-table.jsx

```diff
@@ -41,7 +41,8 @@ import './listing.less';
  * - columns: { title: string, header: boolean, sortable: boolean }[] or string[]
  * - rows: {
  *      columns: (React.Node or string)[],
- *      extraClasses: string[]
+ *      extraClasses: string[],
```

Comment:
The Closes # on this commit has the wrong PR number. Also, the first sentence is very hard to read. I think you mean "If we don't specify an id for each row.."?